pkg/util/log/log.go

```diff
@@ -80,7 +82,8 @@ start:
 	return function, file, lineNumber
 }
 
-func CreateProjectfolder(filePath string) {
+// createLogfolder create the log folder if not exists
+func createLogfolder(filePath string) {
```

Comment:
- Why do you need to `Stat` the file?  Won't `os.MkDirAll` just do the right thing (ignores existing directories and creates all in path).

```suggestion
func createLogFolder(filePath string) {
```